### DISCUSSION TITLE
Package logreq: File 'output.run.xml' not created by logreq.

### DISCUSSION THREAD

**Discussion by Yxiaofa** [nju-lug/NJUThesis#239] [Q&A] [answered]
为啥会突然出现这个错误呀，Package logreq: File 'output.run.xml' not created by logreq.

**Comment by atxy-blip** [comment, marked as answer]
`logreq` 是 `biblatex` 引入的。这个我也没有好的复现手段，建议清除掉所有编译临时文件，从头编译。
